models/particlenet_AK4/config.pbtxt

```diff
@@ -1,38 +1,38 @@
 name: "particlenet_AK4"
 platform: "onnxruntime_onnx"
-max_batch_size : 50
+max_batch_size : 500
 dynamic_batching {
```

Comment:
should there be a preferred batch size here? (and in the other configs below)

Reply:
I changed these in the latest push.  Also based on the benchmarking from the slides referenced above